db/migrate/20240214090208_remove_provider_id_from_mentor_training.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveProviderIdFromMentorTraining < ActiveRecord::Migration[7.1]
+  def change
+    remove_column :mentor_trainings, :provider_id, :uuid
```

Comment:
To avoid coupling mentor trainings to claims, could we keep this field in? This way we can get the context of which mentor trainings happened between which providers and mentors without the need to involve claims made.

Reply:
We can but why did we want to say a claim belongs to a provider? https://github.com/DFE-Digital/itt-mentor-services/pull/231

Right now, we technically have two ways of linking a claim and a provider.

Do you want me to revert this then? https://github.com/DFE-Digital/itt-mentor-services/pull/231

Reply:
A claim creates a requirement of all MentorTrainings be scoped to a single `provider_id` by adding `Claim#provider_id`.

The concept of a `MentorTraining` can be standalone.

Here's a link to our discussion on Slack: https://ukgovernmentdfe.slack.com/archives/C06CUPM2HR8/p1707403465382709

Reply:
OK, I'll keep the provider_id on the mentor training

Reply:
Removed the commit that was removing the provider id